nodes/geometry/bmesh_tesselate.py

```diff
@@ -0,0 +1,43 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types.node import AnimationNode
+
+class BMeshTessellateNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_BMeshTessellateNode"
+    bl_label = "Triangulate BMesh"
+    bl_width_default = 160
+    searchLabels = ["Tesselate BMesh"]
+
+    quad = IntProperty(name = "Quad Method", update = propertyChanged, max = 3, min = 0)
+    ngon = IntProperty(name = "Ngon Method", update = propertyChanged, max = 1, min = 0)
+
+    def create(self):
+        self.newInput("BMesh", "BMesh", "bm")
+        self.newOutput("BMesh", "BMesh", "bm")
+        self.newOutput("Integer List", "Matching Ngon Indices", "ngonIndices", hide = True)
```

Comment:
what is this used for? Do you have an example?
Is there maybe a better (more descriptiv) name?